apps/passport-client/components/shared/SubscriptionNavigation.tsx

```diff
@@ -0,0 +1,43 @@
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+export function SubscriptionNavigation({ to }: { to: string }) {
+  return (
+    <Navigation>
+      <BackLink to={to}>
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          strokeWidth="2"
+          stroke="currentColor"
+          style={{ height: "16px" }}
+        >
+          <path
+            strokeLinecap="round"
+            strokeLinejoin="round"
+            d="M9 15L3 9m0 0l6-6M3 9h12a6 6 0 010 12h-3"
+          />
+        </svg>
```

Comment:
implement this the same way the other svgs are implemented. also make sure this svg is appropriately licensed.

Reply:
Pulling svgs in through an image tag breaks `stroke="currentColor"`, which means hard-coding it in the SVG. I've done it but with the caveat that if we change our styling in CSS we will have to remember to update the hard-colded `stroke` attribute in the SVG.